model/src/oauth/id.rs

```diff
@@ -8,9 +8,13 @@
 //! [`id`]: ../id
 
 use serde::{Deserialize, Serialize};
-use std::fmt::{Display, Formatter, Result as FmtResult};
+use std::{
+    fmt::{Display, Formatter, Result as FmtResult},
+    num::NonZeroU64,
+};
 
 #[derive(Clone, Debug, Default, Deserialize, Eq, Hash, Ord, PartialEq, PartialOrd, Serialize)]
+// *not* a snowflake
```

Comment:
This should be internally documented better; it feels too informal. It should be more descriptive and use proper grammar.